Lib/test/test_genericalias.py

```diff
@@ -1,14 +1,18 @@
 """Tests for C-implemented GenericAlias."""
 
 import unittest
+import pickle
 from collections import (
     defaultdict, deque, OrderedDict, Counter, UserDict, UserList
 )
 from collections.abc import *
 from contextlib import AbstractContextManager, AbstractAsyncContextManager
 from io import IOBase
 from re import Pattern, Match
+from types import GenericAlias
 
+from typing import TypeVar
+T = TypeVar('T')
```

Comment:
I had to put this in the global scope otherwise it would fail pickling :(

Reply:
Yeah, that's fine. I like this better than putting `global T` in the test function. :-)